lib/Authentication/AuthTokens/ShibAuthToken.php

```diff
@@ -146,7 +146,26 @@ private function getAttributesInitToken(){
             return;
 
         }
-
+        else if($idp == 'https://aai-dev.egi.eu/proxy/saml2/idp/metadata.php'){
+            if( empty($_SERVER['epuid'])){
+                die('Did not recieve required epuid attributes from the EGI Dev Proxy Identity Provider to complete authentication, please contact gocdb-admins');
```

Comment:
Should use full name of "eduPersonUniqueID" or at least the right capitalisation of ""ePUID".